### PR TITLE
feat(tools): merge some tools into the `air` CLI tool

### PR DESCRIPTION
**Pull request by monoid**
`air-beautify` and `air-trace` are merged into single `air` tool.  Its crate name is `fluence-air-cli` as otherwise it confilicts with `air` interpreter crate.

### CODE REVIEW

**Comment by nahsi** [comment]
I will update workflows and release-please

File: tools/cli/air/src/trace/mod.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
```
Comment (mikevoronov) [inline tools/cli/air/src/trace/mod.rs:2]:
```suggestion
 * Copyright 2023 Fluence Labs Limited
```

File: tools/cli/air/src/trace/stats/report.rs [hunk 2]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
+use super::super::utils::parse_tracing_duration;
 use super::log_data::{LogKey, LogRecord, Message};
```
Comment (mikevoronov) [inline tools/cli/air/src/trace/stats/report.rs:18]: let's unwrap imports in this file

**Review by mikevoronov — APPROVED** [review]
utACK

**Review by raftedproc — APPROVED** [review]
utACK

**Review by nahsi — APPROVED** [review]

**Pull request merged**